Adapters remote tests not working

### Describe the bug

The remote adapters tests are not working

See full log in https://keycloak-jenkins/job/universal-test-pipeline-adapters/1095/execution/node/3295/log/

### Version

18.0.0

### Expected behavior

_No response_

### Actual behavior

```
[ERROR] Errors: 
[ERROR]   RemoteOIDCAdapterTest » Resolution Unable to get version for dependency specified by org.keycloak.testsuite:integration-arquillian-testsuite-providers-deployment:compile, it was not provided in neither <dependencyManagement> nor <dependencies> sections.
[ERROR]   RemoteOIDCSessionAdapterTest » Runtime Arquillian initialization has already been attempted, but failed. See previous exceptions for cause
[ERROR]   RemoteSAMLAdapterTest » Runtime Arquillian initialization has already been attempted, but failed. See previous exceptions for cause
[ERROR]   RemoteSAMLFilterAdapterTest » Runtime Arquillian initialization has already been attempted, but failed. See previous exceptions for cause
[INFO] 
[ERROR] Tests run: 4, Failures: 0, Errors: 4, Skipped: 0
```

### How to Reproduce?

_No response_

### Anything else?

_No response_

--- a/testsuite/integration-arquillian/tests/base/src/test/resources/jboss-deployment-structure.xml
+++ b/testsuite/integration-arquillian/tests/base/src/test/resources/jboss-deployment-structure.xml
@@ -29,6 +29,9 @@
             <module name="org.keycloak.keycloak-server-spi"/>
             <module name="org.keycloak.keycloak-server-spi-private"/>
             <module name="org.keycloak.keycloak-services" services="import"/>
+            <module name="org.keycloak.keycloak-model-legacy"/>
+            <module name="org.keycloak.keycloak-model-legacy-private"/>
+            <module name="org.keycloak.keycloak-model-legacy-services"/>
             <module name="org.keycloak.keycloak-model-infinispan"/>
             <module name="org.keycloak.keycloak-model-jpa"/>
             <module name="org.keycloak.keycloak-ldap-federation"/>

